extensions/typescript-language-features/package.json

```diff
@@ -9,7 +9,9 @@
   "license": "MIT",
   "aiKey": "AIF-d9b70cd4-b9f9-4d70-929b-a071c400b217",
   "enableProposedApi": true,
-  "requiresWorkspaceTrust": "onDemand",
+  "workspaceTrust": {
+    "require": "onDemand"
```

Comment:
idk why but I think I would prefer this to be called `required` instead of `require`

Reply:
Fixed.